LondonBritishLibrary/orient/BLorient594.xml

```diff
@@ -0,0 +1,466 @@
+                                    <material key="parchment"/>
+                                    <p>The last folio is mutilated.</p>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">198</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>10.5</height>
+                                        <width>9.125</width>
+                                    </dimensions>
+                                </extent>
+
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="20 21"/>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>The manuscript is written in fine hand, of the <date>17th century.</date></desc>
+                              <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+                            </handNote>
+
+                        </handDesc>
+                        <decoDesc>
+                          <summary>Pictures.</summary>
+                            <decoNote xml:id="d1" type="drawing">
+                              <locus target="#177v"></locus>
+                              <desc>The blessed Virgin and child.</desc>
+                              </decoNote>
+
+                            <decoNote xml:id="d2" type="drawing">
+                              <locus target="#141v"></locus>
+                              <desc>The crucifixion.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                          <list>
+
+                            <item xml:id="a1">
+                              <locus target="#3r"></locus>
+                              <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                              <q xml:lang="gez">አርጋኖን፡ ዘ<placeName ref="INS0101MadhaneAlam">
+                                ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q></item>
+
+                                <item xml:id="a2">
+                                <locus target="#197r"/>
+                                <desc type="GuestText"><title ref="LIT3984basamay">Hymn to the blessed Virgin Mary</title></desc>
+                                <q xml:lang="gez"><seg part="I">በሰማይ፡ ወምድር፡ አልብየ፡ ባዕደ፡ አበ፡ ወእመ፡ እኅተ፡ ወውሉደ፡ ማርያም፡ ድንግል፡ እትአመነኪ፡ ገሃደ፡</seg></q>
+                                </item>
+
+<item xml:id="a3">
+<locus target="#1r"/>
+<desc>Significations of the letters of the alphabet.</desc>
+<q xml:lang="gez">ፊደል፡ ሀ፡ ብሂል፡ ሀልዎቱ፡ ለአም፡ እምቅድመ፡ ዓለም።</q>
+</item>
```

Comment:
I think this one is https://betamasaheft.eu/works/LIT3975FekkareF/main. Please do not forget to encode the type (GuestText), and encode seg if the provided text is only an incipit.